.github/workflows/build.yaml

```diff
@@ -56,6 +56,13 @@ jobs:
         go-version: ${{ matrix.golang_version }}
     - name: Install Python dependencies
       run: pip install codecov tox tox-gh-actions
+    - name: Download google-java-format.jar
```

Comment:
Please do not download the artifact in github actions.

To test the change we only need to verify that we are invoking the shell command using the provided path for the JAR file.
This implies a simple and fast unit test.